### PR TITLE
Use Resource Consumer for tests in autoscaling.go

### PR DESCRIPTION
**Pull request by socaa**
Part of #11570 

@piosz

### CODE REVIEW

File: test/e2e/autoscaling.go [hunk 1]
```diff
@@ -140,39 +142,6 @@ func CreateService(f *Framework, name string) {
 	Expect(err).NotTo(HaveOccurred())
 }
 
-func ConsumeCpu(f *Framework, id string, cores int) {
-	CreateService(f, id)
```
Comment (piosz) [inline test/e2e/autoscaling.go]: You can also remove CreateService method.

**Comment by piosz** [comment]
LGTM just a nit

**Comment by socaa** [comment]
PTAL

@piosz

**Comment by piosz** [comment]
LGTM

**Pull request merged**